.gitignore

```diff
@@ -1,2 +1,5 @@
 node_modules
 dist
+package-lock.json
+package-lock.json.*
+yarn.lock
```

Comment:
I typically track `yarn.lock` (it is already checked in) since it ensures consistent installs across machines so I think we need to drop this line. From https://yarnpkg.com/lang/en/docs/yarn-lock/#toc-check-into-source-control

> All yarn.lock files should be checked into source control (e.g. git or mercurial). This allows Yarn to install the same exact dependency tree across all machines, whether it be your coworker’s laptop or a CI server.
>
>Framework and library authors should also check yarn.lock into source control. Don’t worry about publishing the yarn.lock file as it won’t have any effect on users of the library.

Reply:
> I typically track yarn.lock (it is already checked in)

Ah, cool, cool. I should have checked before I added this file to the gitignore. I will remove this from the gitignore :v: